### PR TITLE
Update HappyNewYear.cpp

### PR DESCRIPTION
**Pull request by jimmyMotU**
trying to add my little piece of code...

### CODE REVIEW

File: HappyNewYear.cpp [hunk 1]
```diff
@@ -35,20 +34,32 @@ int main()
...
-            
+
         srand(time(0));
         noMoreCovid=rand() %2; // mostly random, as the covid forecasts
-       
-        if (noMoreCovid || vaccinationDone )    cout<<"Book your next vacation flight NOW\n";
+
+        cout<< "Are you fully vaccinated now? Answer 0 for No or your favorite number for Yes : ";
+        cin>> vaccinationDone; cin.clear();
+        if ( vaccinationDone > 1 ) vaccinationDone = true;
```
Comment (Fab20100) [inline HappyNewYear.cpp:48]: Shouldn't it be >= 1?

Comment (jimmyMotU) [inline HappyNewYear.cpp:48]: As true is 1 and false is 0, when user enters 0 or 1 all is fine. If users enters something else we set boolean value to true, reason why >1 is sufficient here. On top of that, bool is of type 'unsigned int' so no need to test negative values. By the way we could remove totally the line 48 as anything else than 0 is by default true, I just wished to give back to Cesar what belongs to Cesar, means limit the value to true or false to a bool type variable.
